src/runtime/test/plan/planificator-test.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
.ts tests work it appears.

Reply:
so should i have just named the new test file .ts, no other special steps needed? i think i'll just push for now with all the changes going in, and convert later.